content/en/api/integrations_pagerduty/pagerduty_create_code.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Create a PagerDuty integration
+type: apicode
+order: 18.2
+external_redirect: /api/#create-a-pagerduty-integration
+---
+
+##### Signature
+`PUT https://api.datadoghq.com/api/v1/integration/pagerduty`
```

Comment:
shouldn't this be a POST?

Reply:
This is what i thought too.. but no this is a `PUT` after testing idk why.